submissions/TaniaBondarenko/html-movie-seat-booking/index.html

```diff
@@ -13,140 +13,105 @@
           </div>
 
           <div class="badgeSeat vip"></div>
           <div class="ticketType">
-            <div class="name"><a>Vip</a></div>
+            <div class="name">Vip</div>
             <span class="price">300</span>
             <small class="currency">usd</small>
           </div>
         </div>
         <div class="legend">
           <div class="occupiedSeat"></div>
-          <div class="description">
-            <a>Occupied<br />seat</a>
-          </div>
+          <div class="description">Occupied seat</div>
           <div class="blockedSeat"></div>
-          <div class="description">
-            <a>Blocked<br />seat</a>
-          </div>
+          <div class="description">Blocked seat</div>
         </div>
       </section>
 
       <div class="cinemaContainer">
         <div class="filmInfo">
           <h2 class="filmTitle">2001: A Space Odyssey</h2>
-          <br />
           <p>2h 29min</p>
           <p>Adventure, Sci-Fi</p>
           <p>IMDb 8.3</p>
-          <time datetime="1968-05-12"> 12 May 1968</time><br />
+          <time datetime="1968-05-12"> 12 May 1968</time>
         </div>
         <div class="screenImg">
           <svg xmlns="http://www.w3.org/2000/svg" viewBox="-290 -20 1400 50">
-            <path
-              d="M3.2,20l-2,0.2l-0.3-4l2-0.2C136.2,5.3,269.6,0,403,0s266.8,5.3,400.2,16l2,0.2l-0.3,4l-2-0.2 C669.5,9.3,536.3,4,403,4S136.4,9.3,3.2,20z"
-            ></path>
+            <path d="M3.2,20l-2,0.2l-0.3-4l2-0.2C136.2,5.3,269.6,0,403,0s266.8,5.3,400.2,16l2,0.2l-0.3,4l-2-0.2 C669.5,9.3,536.3,4,403,4S136.4,9.3,3.2,20z"></path>
           </svg>
         </div>
         <div class="screenText">SCREEN</div>
 
         <div class="seats">
-          <br />
           <section class="rowLux">
             <span class="rowText">1</span>
             <input id="checkbox1-6" type="checkbox" value="1-6" class="hiddenCheckbox" tabindex="-1" />
-            <button class="seat"></button>
-            <label for="checkbox1-6"></label>
+            <label class="seatLabel" for="checkbox1-6"><span class="seat" tabindex="0"></span></label>
             <input id="checkbox1-5" type="checkbox" value="1-5" class="hiddenCheckbox" tabindex="-1" />
-            <button class="seat"></button>
-            <label for="checkbox1-5"></label>
-            <input id="checkbox1-4" type="checkbox" value="1-4" title="The seat is blocked" class="hiddenCheckbox disabled notActive" tabindex="-1" />
-            <button class="seat" disabled title="The seat is blocked"></button>
-            <label for="checkbox1-4"></label>
-            <input id="checkbox1-3" type="checkbox" value="1-3" title="The seat is blocked" class="hiddenCheckbox disabled notActive" tabindex="-1" />
-            <button class="seat" disabled title="The seat is blocked"></button>
-            <label for="checkbox1-3"></label>
+            <label class="seatLabel" for="checkbox1-5"><span class="seat" tabindex="0"></span></label>
```

Comment:
You don't need `tabindex="0"` here, you can receive focus from the input.

Reply:
Please clarify about input. Because when I replaced `tabindex="0"` there was no any focus on my seats. That's why I've added it.

Reply:
@TaniaBondarenko  Watch [this video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=528s) about checkboxes.